src/components/dev-hub/tab.js

```diff
@@ -10,6 +10,9 @@ const Tab = styled('div')`
     border-bottom: 1px solid ${colorMap.greyDarkOne};
     display: flex;
     justify-content: space-between;
+    @media ${screenSize.upToLarge} {
```

Comment:
This works! Didn't even think of just removing flexbox :)

Can break at `medium` here instead of `large` since that is where most of our mobile styles kick in (nav, single card per row, etc) and below?